setup.cfg

```
@@ -1,8 +1,46 @@
 [bdist_wheel]
 universal = 1
 
+[flake8]
+max-line-length=200
+
 [metadata]
+name = voila-gridstack
+version = 0.2.0
+description = A GridStack template for Voila.
+long_description = file: README.md
+long_description_content_type = text/markdown
 license_file = LICENSE
+author = Voila Development Team
+url = https://github.com/voila-dashboards/voila-gridstack
+platforms = Linux, Mac OS X, Windows
+keywords = Jupyter, JupyterLab, Voila, voila-gridstack, gridstack
+classifiers =
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Framework :: Jupyter
+    Framework :: Jupyter :: JupyterLab
+    Framework :: Jupyter :: JupyterLab :: 3
+    Framework :: Jupyter :: JupyterLab :: Extensions
+    Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt
 
-[flake8]
-max-line-length=200
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+python_requires = >=3.6
+install_requires =
+    jupyterlab_widgets~=1.0
+    voila>=0.2.0,<0.3.0
+
+[options.extras_require]
+test =
+    ipykernel
+		pytest
+		pytest-tornasync
+		lxml
```

Minor formatting edit:

```suggestion
    ipykernel
    pytest
    pytest-tornasync
    lxml
```